src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
+
+The STEM acronym itself has come under some criticism for excluding art and design, leading to initiatives such as the <a href="http://stemtosteam.org/" target="_blank" class="link-external">STEM to STEAM Initiative</a>.
+
+We encourage those considering hacking and making in learning contexts to be aware of larger issues of diversity in STEM subjects, workplaces and communities, as these issues can have relevance and knock-on effects to the diversity and inclusion of hackathons and hacking.
+
+## Tools
+
+The sections below include examples of tools (software, hardware and design-related) that can be useful in inclusive design hacking. Some are early-stage research projects, while others are mature and in widespread usage in and outside the classroom.
+
+We have tried to highlight tools that fall somewhere at the nexus of openness, active development, strong communities, and adaptability. Separately or in combination, they might be used to:
+
+* prototype new ways of interacting with the digital or physical world that suit an individual's particular needs.
+* adapt and transform existing technology to support new uses.
+
+### Research From Prosperity 4All
+
+Floe participates in the <a href="http://www.prosperity4all.eu/" target="_blank" class="link-external">Prosperity 4All</a> project that has supported the development and cataloguing of tools relevant to inclusive learning through hacking and making. These include:
+
+#### The Nexus
+
+The <a href="https://github.com/GPII/nexus" target="_blank" class="link-external">Nexus</a> is an integration technology based on the <a href="https://github.com/fluid-project/infusion" target="_blank" class="link-external">Infusion</a> Javascript framework that allows <a href="https://wiki.gpii.net/w/Nexus_API" target="_blank" class="link-external">distributed systems to be connected together through shared and transformed state over HTTP and WebSockets protocols, as described in detail here.</a>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/7R_pz2Fz4qE" frameborder="0" allowfullscreen></iframe>
+
+#### AsTeRICS
+
+The <a href="https://github.com/asterics" target="_blank" class="link-external">Assistive Technology Rapid Integration &amp; Construction Set</a> is an open source construction set for assistive technologies.
+
+AsTeRICS can be used to rapidly prototype new forms of assistive technology using a graphical construction environment that runs on Windows or as a <a href="https://github.com/asterics/WebACS" target="_blank" class="link-external">web-based application.</a>
```

Comment:
An example might be cool here. I know the Austrian teams have run several hacking-type events with AsTeRICs—I wonder if we can include that somehow?